### PR TITLE
Subtask/core 10398 update open api package

### PR DESCRIPTION
**Pull request by WyattGrant**
## Description

Adds method for new open api endpoint to retrieve wait times.

## Motivation and context

a vendor needed this for a custom view they're making.

## How has this been tested?

wrote a js test and tested it locally.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

Please, please, please, don't send your pull request until all of the boxes are ticked. Once your pull request is created, it will trigger a build on our [continuous integration](https://travis-ci.org) server to make sure your [tests and code style pass](https://help.github.com/articles/about-required-status-checks/).

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [x] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

File: src/resources/wait-time.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { AxiosInstance } from 'axios';
+
+import Conditional from './conditional';
+
+export interface WaitTimeParameters {
+  location_id?: string | number;
```
Comment (craigpaul) [inline src/resources/wait-time.ts:6]: Can we change this to simply `location` in order to stick to the pre-set convention

File: src/resources/wait-time.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    if (this.limit) {
+      params.limit = this.limit;
+    }
+
+    if (this.page) {
+      params.page = this.page;
+    }
+
+    return await this.client.get(`wait-time-average/${location_id}`.replace(/\/$/, ''), { params });
```
Comment (craigpaul) [inline src/resources/wait-time.ts:48]: What is the purpose of the regex here?

**Review by craigpaul — CHANGES_REQUESTED** [review]

File: src/resources/wait-time.ts (hunk 2 above)
Comment (WyattGrant) [reply src/resources/wait-time.ts:48]: just removes the slash at the end, if there is one

Comment (craigpaul) [reply src/resources/wait-time.ts:48]: Is it actually required? I'm not sure why that would need to be in there

Comment (WyattGrant) [reply src/resources/wait-time.ts:48]: definitely not required, i just didn't like seeing the trailing slash haha. i can remove it.

**Review by craigpaul — APPROVED** [review]

**Pull request merged**